src/caselawclient/models/utilities/aws.py

```diff
@@ -208,6 +208,13 @@ def announce_document_event(uri: DocumentURIString, status: str, enrich: bool =
     )
 
 
+def upload_asset_to_private_bucket(body: bytes, s3_key: str) -> None:
+    """Upload an asset to the private bucket."""
+    bucket: str = env("PRIVATE_ASSET_BUCKET")
+    s3client = create_s3_client()
+    s3client.put_object(Body=body, Bucket=bucket, Key=s3_key, Tagging="pdfsource=custom-pdfs")
```

Comment:
I'm not 100% on the `Tagging` here. I agree it should be something, but I'm wondering if it would be better as something like `manual-upload-via-{{source}}` where `{{source}}` is an argument we pass.